.github/workflows/build.yml

```diff
@@ -109,3 +109,12 @@ jobs:
           path: |
             ./dist/${{ matrix.target }}/thyra-installer
             ./dist/${{ matrix.target }}/thyra-installer.exe
+      - name: Create release and upload binaries
```

Comment:
I couldn't find the information in the official documentation, but here is a basic explaination:
> The nice thing about softprops/action-gh-release is that you can keep adding artifacts to the same release as long as the workflow run is triggered by a push to the same tag.
> https://stackoverflow.com/a/66959862

So, since we are using the same tag, we can add artifacts without creating a new release each time. It's what we are already doing with the binaries [here](https://github.com/massalabs/thyra/blob/d39f6b05ebe71fc6d10a16c578bccbbe76a55bc0/.github/workflows/build.yml#L69)

Reply:
awesome!